miasm2/jitter/jitcore.py

```diff
@@ -160,7 +161,7 @@ def jit_call(self, label, cpu, vmmngr):
         """
 
         fc_ptr = self.lbl2jitbloc[label]
-        return self.exec_wrapper(fc_ptr, cpu.cpu, vmmngr.vmmngr)
+        return self.exec_wrapper(fc_ptr, cpu)
```

Comment:
Yes, its related, but I am not quite used to manipulate LLVM, nor its code generator.
So this part is let for another fix